woocommerce/woocommerce-gateway-stripe
# Show currency requirement details in a popover

Fixes #2970 

## Changes proposed in this Pull Request:
Implemented the new design for the popover linked to the currency requirement pill. (design component Kofpo8jj4zDWXDsVdlSz0f-fi-6795%3A10109)

**Before**

![before pop](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/35a01dfe-d4b0-47f8-8339-38e7a3c1cbc8)

**After**

![after pop](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/1e61993f-6a44-4342-8c1a-96ed60ab454c)


## Testing instructions
- Set store currency to USD.
- Go to the Stripe settings page and check the payment methods.
- Confirm that some of the payment methods like giropay, EPS, OXXO, Boleto etc. have a pill beside their name `Requires currency`.
- Click on the `info` icon in the currency requirement pill and confirm that a popover is displayed. The popover should have the new design. 
- Click outside and the popover should be hidden.

![popup](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/ff285b05-b714-4b4b-b0a7-fa4d6de086f5)
